extension/src/popup/views/Account/index.tsx

```diff
@@ -273,7 +284,7 @@ export const Account = () => {
                   />
                 </div>
               )}
-              {!isFunded && !hasError && (
+              {balances !== null && !isFunded && !hasError && (
```

Comment:
@piyalbasu I'd like to chat through why this check is needed because I still think it should not be needed but the rendering steps still cause this to flash in between account changes without it, the state at that point doesn't make a ton of sense to me yet but this check avoids the flash because `balances` is only null before fetching balances or in the event of an error, which both are cases where this should not be shown.